### PR TITLE
Make stake, profit and buy button inline

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

File: src/_common/M.js [hunk 1]
```diff
@@ -15,11 +15,13 @@ export default class M extends Component {
 		const { m = '', values } = this.props;
 
 		return (
-			<FormattedMessage
-				id={m}
-				defaultMessage={m}
-				values={values}
-			/>
+			<span {...this.props}>
```
Comment (borisyankov) [inline src/_common/M.js]: What is the purpose of this?

Comment (qingweibinary) [inline src/_common/M.js]: To be able to style M, like font-weight etc

Comment (borisyankov) [inline src/_common/M.js]: That was one of my annoyances that they don't pass a className down to their span, but they discussed it on the Issues part of GitHub and was against it. So this is probably better.

Comment (qingweibinary) [inline src/_common/M.js]: Yes, I read that and thus select this method, they have an internal id, which is not html id at all

**Pull request merged**